src/olympia/devhub/views.py

```diff
@@ -307,9 +309,15 @@ def feed(request, addon_id=None):
 
 @dev_required
 def edit(request, addon_id, addon):
+    try:
+        whiteboard = Whiteboard.objects.get(pk=addon.pk)
+    except Whiteboard.DoesNotExist:
+        whiteboard = Whiteboard(pk=addon.pk)
```

Comment:
This doesn't create the `Whiteboard` instance, I think you want to either do a `whiteboard.save()` afterwards or use https://docs.djangoproject.com/en/1.8/ref/models/querysets/#django.db.models.query.QuerySet.get_or_create?